views/incorrect-information/wrong-ro.html

```diff
@@ -9,24 +9,105 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <form action="" method="post">
-        <h1 class="govuk-heading-xl">
-          You need to update the company details
-        </h1>
-        <p>You will need to
-          <a href= "{{changeRoaUrl}}" target="_blank" rel="noopener noreferrer" data-event-id="update-roa-link">update the registered office address (opens in a new tab)</a>
-          before you can file a confirmation statement.</p>
-        {{ govukInsetText({
-            text: "An update to the registered office address can take some time to process, so it may not be reflected immediately in the information displayed on this service. As long as the update has been submitted, you can still file the confirmation statement."
-        }) }}
-        {{ govukButton({
-          text: "Return to confirmation statement",
-          href: taskListUrl,
-          attributes: {
-            "data-event-id": "return-to-confirmation-statement-button"
-          }
-        }) }}
-        <p><a href="https://www.smartsurvey.co.uk/s/fileconfstmnt-wrongro/">What did you think of this service?</a>
-        (takes 2 minutes)</p>
+        <main class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content" role="main">
+
+          <h1 class="govuk-heading-l">
+            You need to update the company details
+          </h1>
+
+          <ol class="app-step-nav__steps">
+            <li class="app-step-nav__step js-step step-is-shown" id="check-youre-allowed-to-drive">
```

Comment:
these id's will have to be checked with UX, looks lie they've come from the dvla and we can't use them in our code. The aria ones will be read by screen readers
 id="check-youre-allowed-to-drive"
aria-controls="step-panel-driving-lessons-and-practice-3"/>
id="get-a-provisional-licence"
etc...